contentcuration/contentcuration/views.py

```diff
@@ -14,7 +14,7 @@
 from kolibri.content.models import File
 
 def base(request):
-    return redirect(channel_list)    # redirect to the channel list page
+    return redirect('channels')    # redirect to the channel list page
```

Comment:
Shouldn't this be `reverse("channels")`? This looks like it might work by accident, since the url slug is also "channels".